Resources/Prototypes/Nyanotrasen/Entities/Clothing/psionic_clothing.yml

```diff
@@ -64,3 +64,19 @@
   - type: ClothingGrantPsionicPower
     power: NoosphericZapPower
   - type: Psionic
+
+- type: entity
+  parent: FelinidCone
+  id: felinidConePsionic
+  name: ZE CONE OF ALL-SEEING
+  description: CONE CONE!!!?!??!?!??!?!!
+  abstract: false
+  components:
+  - type: IngestionBlocker
+  - type: ClothingGrantPsionicPower
+    power: TelegnosisPower
+  - type: Psionic
```

Comment:
If you're going to make psionic clothing, you should probably put it in the PsionicArtifactPool.